src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const REMOVED_BOOK = 'REMOVE_BOOK';
+const ADD_BOOK = 'ADD_BOOK';
+
```

Comment:
- [ ] You are required to use the ducks' pattern for your action types. See example below:

  ```
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  ```